src/electionguard_gui/models/key_ceremony_dto.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, List
+from datetime import datetime
+from electionguard.key_ceremony import ElectionPublicKey
+
+from electionguard_gui.eel_utils import utc_to_str
+from electionguard_gui.services.db_serialization_service import (
+    dict_to_election_public_key,
+)
+
+# pylint: disable=too-many-instance-attributes
+class KeyCeremonyDto:
+    """A key ceremony for serializing to the front-end GUI and providing helper functions to Python."""
```

Comment:
This isn't strictly a DTO, it's more of a sort of intermediary class that helps rehydrate key ceremony data from the database and then help get that data to the front-end.  I'd welcome a better name for this class, else I'll eventually arrive at something.  This architecture is dynamic and quickly evolving.